server/src/main/java/edp/core/utils/SqlUtils.java

```diff
@@ -505,6 +505,11 @@ private String getDBSchemaPattern(String schema) {
             case SQLSERVER:
                 schemaPattern = "dbo";
                 break;
+            case CLICKHOUSE:
+                if (!StringUtils.isEmpty(schema)) {
+                    schemaPattern = schema;
+                }
+                break;
             case PRESTO:
```

Comment:
The code is duplicate in the PRESTO case, you can fix it like this:
```java
case CLICKHOUSE:
case PRESTO:
    if (!StringUtils.isEmpty(schema)) {
        schemaPattern = schema;
    }
    break;
```
and resubmit it.

Reply:
ok,i had resubmited.thx